product/quickstart/desktop-and-mobile.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: "Desktop and mobile"
+description: "Learn to build responsive designs that look good across desktop and mobile devices."
+icon: "laptop-mobile" iconType: "solid"
+---
+
+So far, you've only viewed your site on desktop, but Makeswift allows you to preview your site on sizes that simulate desktop, tablet, and mobile. Before previewing on different devices, take another look at the Hero component.
```

Comment:
Sending someone from the Product Quickstart to the Developer docs for custom breakpoints would be a bit jarring since it jumps directly into a code snippet which isn't covered in the Quickstart. Going to hold off on this for now.